sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Microsoft.Azure.Management.ServiceBus.csproj

```diff
@@ -7,11 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.ServiceBus</PackageId>
 		<Description>Provides developers with libraries to create and manage Namespaces and manage Authorization Rules. Note: This client library is for ServiceBus under Azure Resource Manager.</Description>
 		<AssemblyName>Microsoft.Azure.Management.ServiceBus</AssemblyName>
-		<Version>4.0.0</Version>
+		<Version>4.1.0</Version>
```

Comment:
Since there are a lot breaking changes and some of them cannot be worked around, you need to bump the version to `5.0.0`.

Reply:
updated API version